raccoongang/edx-platform
# Baobab 41 add external registration link

**Description:**
Hide registration button and on the direct link redirect to the example.com

**Youtrack:** 
https://youtrack.raccoongang.com/agiles/104-5/105-5?issue=BAOBAB-41


**Configuration instructions:** 
Setup next flag:
FEATURES['ENABLE_EXTERNAL_REGISTRATION'] = True
FEATURES['ALLOW_PUBLIC_ACCOUNT_CREATION'] = False
EXTERNAL_REGISTRATION = 'https://example.com' #for now it is example.com 

**Testing instructions:**
Open platform. There no registration button here. 
Go to `/register` - example.com appears. 

**Reviewers:**

**Related Confluence documents:**
- https://raccoongang.atlassian.net/wiki/spaces/RG/pages/109543450/Baobab+DevOps+Info

**Post merge:**
